templates/unstable-vite-cloudflare/vite.config.ts

```diff
@@ -0,0 +1,21 @@
+import { unstable_vitePlugin as remix } from "@remix-run/dev";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// Check out .cloudflare/README.md for why there's a `.cloudflare` directory in this template
+// In the future, you'll be able to import the `cloudflare` adapter from `@remix-run/cloudflare`
+import { adapter as cloudflare } from "./.cloudflare";
+
+export default defineConfig({
+  ssr: {
+    resolve: {
+      externalConditions: ["workerd", "worker"],
+    },
+  },
```

Comment:
TODO: this should be set automatically by the `cloudflare` adapter

Reply:
from chatting w/ Mark: lets do `{ remixConfig, viteConfig }` as the return shape for adapter config